apis/extension/node_resource_amplification.go

```diff
@@ -130,6 +130,12 @@ func SetNodeRawCapacity(node *corev1.Node, capacity corev1.ResourceList) {
 	node.Annotations[AnnotationNodeRawCapacity] = string(s)
 }
 
+// HasNodeRawAllocatable checks if the node has raw allocatable annotation.
+func HasNodeRawAllocatable(annotations map[string]string) bool {
```

Comment:
Add a simple test for this new function?

Reply:
will be covered elsewhere